apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam._index/route.tsx

```diff
@@ -245,16 +245,14 @@ export default function Page() {
                     </div>
                     <Table containerClassName="max-h-full pb-[2.5rem]">
                       <TableHeader>
-                        <TableRow>
-                          <TableHeaderCell>Task ID</TableHeaderCell>
-                          <TableHeaderCell>Task</TableHeaderCell>
-                          <TableHeaderCell>Running</TableHeaderCell>
-                          <TableHeaderCell>Queued</TableHeaderCell>
-                          <TableHeaderCell>Activity (7d)</TableHeaderCell>
-                          <TableHeaderCell>Avg. duration</TableHeaderCell>
-                          <TableHeaderCell>Environments</TableHeaderCell>
-                          <TableHeaderCell hiddenLabel>Go to page</TableHeaderCell>
-                        </TableRow>
+                        <TableHeaderCell>Task ID</TableHeaderCell>
+                        <TableHeaderCell>Task</TableHeaderCell>
+                        <TableHeaderCell>Running</TableHeaderCell>
+                        <TableHeaderCell>Queued</TableHeaderCell>
+                        <TableHeaderCell>Activity (7d)</TableHeaderCell>
+                        <TableHeaderCell>Avg. duration</TableHeaderCell>
+                        <TableHeaderCell>Environments</TableHeaderCell>
+                        <TableHeaderCell hiddenLabel>Go to page</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"